documentation/docs/07-a-options.md

```diff
@@ -16,30 +16,28 @@ We can mitigate that by _prefetching_ the data. Adding a `sveltekit:prefetch` at
 
 ...will cause SvelteKit to run the page's `load` function as soon as the user hovers over the link (on a desktop) or touches it (on mobile), rather than waiting for the `click` event to trigger navigation. Typically, this buys us an extra couple of hundred milliseconds, which is the difference between a user interface that feels laggy, and one that feels snappy.
 
-> `sveltekit:prefetch` is a SvelteKit idiom, not a standard attribute for `<a>` elements
-
 You can also programmatically invoke `prefetch` from `$app/navigation`.
 
-### rel=external
+### sveltekit:noscroll
 
-By default, the SvelteKit runtime intercepts clicks on `<a>` elements and bypasses the normal browser navigation for relative (same-origin) URLs that match one of your page routes. We sometimes need to tell SvelteKit that certain links need to be be handled by normal browser navigation.
+When navigating to internal links, SvelteKit mirrors the browser's default navigation behaviour: it will change the scroll position to 0,0 so that the user is at the very top left of the page (unless the link includes a `#hash`, in which case it will scroll to the element with a matching ID).
 
-Adding a `rel=external` attribute to a link...
+In certain cases, you may wish to disable this behaviour. Adding a `sveltekit:noscroll` attribute to a link...
 
 ```html
-<a rel="external" href="path">Path</a>
+<a href="path" sveltekit:noscroll>Path</a>
 ```
 
-...will trigger a browser navigation when the link is clicked.
+...will prevent scrolling after the link is clicked.
 
-### sveltekit:noscroll
+### rel=external
```

Comment:
honestly i was slightly surprised to find this in the docs, i assumed that decision had been made elsewhere. i prefer `target=_self` personally, `rel=external` seems incorrect

Reply:
I'd probably be in favor of just mentioning `target` here and completely removing the `rel=external` handling from the click interceptor. Unless that would be more confusing? I know we stole a fair bit of this code from page.js (including, it looks like, the `rel=external` thing), but I don't know whether that was for convenience or because someone considered its behavior to be a de facto standard for click-interception in SPAs.

Reply:
To be fair, `target=_self` is also kinda confusing. It's not obvious to me why that would result in a page refresh. Maybe we remove handling for both things and add a `sveltekit:refresh` attribute?

Reply:
That I'm not totally comfortable with, because if someone had a link with (say) `target=_blank`, then it'd be weird to not respect that by default and to require people to also have `sveltekit:refresh`. Although that could partially be an artifact of me being used to how click-handler-based SPAs generally work.

Reply:
Or, similarly, if someone had a `download` attribute on a link, but we just plowed through with the click interception unless they also had `sveltekit:refresh`, that also feels weird.

Reply:
`rel=external` seems like an overloaded usage to me. E.g. you might have an app on same domain (though possibly different subdomain) that's still yours or part of the site that shouldn't be tagged with `rel=external`. `target=_self` seems better though still possibly overloaded. I think that we could respect `rel=external`, `target`, etc. as being external but it might be nice to have something like `sveltekit:external` to be able to specify it without having to use those things which already have other meaning and might not be applicable

Reply:
https://github.com/sveltejs/kit/issues/546

Reply:
I'll leave this bit of the docs as they are for now, until we can resolve #546